components/src/preact/lineageFilter/lineage-filter.tsx

```diff
@@ -85,15 +90,32 @@ const LineageFilterInner: FunctionComponent<LineageFilterInnerProps> = ({
 
     return (
         <>
-            <input
-                type='text'
-                class='input input-bordered w-full'
-                placeholder={placeholderText !== undefined ? placeholderText : lapisField}
-                onInput={onInput}
-                ref={inputRef}
-                list={lapisField}
-                value={initialValue}
-            />
+            <div className='relative w-full'>
+                <input
+                    type='text'
+                    class='input input-bordered w-full pr-10'
+                    placeholder={placeholderText !== undefined ? placeholderText : lapisField}
+                    onInput={onInput}
+                    ref={inputRef}
+                    list={lapisField}
+                    value={inputValue}
+                />
+                {hasInput && (
+                    <button
+                        type='button'
+                        name='✕'
+                        className='absolute top-1/2 right-2 transform -translate-y-1/2 text-gray-500 hover:text-gray-700'
+                        onClick={() => {
+                            if (inputRef.current) {
+                                inputRef.current.value = '';
+                                onInput();
+                            }
+                        }}
```

Comment:
I don't like the side effects of this - I' propose to implement it as:
```suggestion
                        onClick={() => onInput(undefined)}
```
(see other comments for how to adapt the rest)